SpringProject/kinematic_controller_solver_test.m

```diff
@@ -124,6 +124,7 @@
 phi_d = pi/2.*ones(1,length(x_d));
 theta_d =  zeros(1,length(x_d));
 psi_d =  zeros(1,length(x_d));
+state = zeros(1, 36);
```

Comment:
switch indexing dimensions

Reply:
Fixed in c0609a5fd791bf6ff06dc30c0050992cd16cbaa7